roles/traefik/templates/traefik_dynamic.yaml

```diff
@@ -47,7 +47,11 @@ http:
       stripPrefix:
         prefixes:
           - "/gateway"
-
+    # This is a workaround due to a mismatch between the keycloak realm name and the nebari-jupyterhub-theme
+    realm-replacepathregex:
+      replacePathRegex:
+        regex: "^/auth/admin/nebari/console/"
+        replacement: "/auth/"
```

Comment:
It would have been better to replace the whole url instead of redirecting to just `/auth`, as now the user have to click another link on the admin page to go to the keycloak admin page, but since that might require change in `redirect_uri` and also we're in a crux of time, I'll pass.